locales/FACTER.pot

```diff
@@ -430,7 +430,7 @@ msgid ""
 "mismatched processor frequencies found; facter will only report one of them"
 msgstr ""
 
-#. debug
```

Comment:
.pot files should not be merged up since they are generated automatically by the CI